components/Tabs/src/TabBarView/MDCTabBarViewUnderlineIndicatorTemplate.h

```diff
@@ -0,0 +1,23 @@
+// Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
```

Comment:
Please fix year

Reply:
This was a copy-paste, so @jverkoey says don't change copyright.